layouts/partials/header.html

```diff
@@ -14,9 +14,41 @@
         menu
       </button>
       <div>
+        {{/* main logo */}}
         <a href="{{ site.BaseURL }}">
           <div>
-            {{ readFile "static/assets/images/docker-docs-logo.svg" | safeHTML }}
+            <svg width="92" height="33" viewBox="0 0 92 33" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
A tip: If you want to inline SVGs, I would recommend putting (or mounting) them inside `assets` and then use `(resources.Get "foo.svg").Content | safeHTML`

Reply:
ah that is nice. I inlined it here because keeping it in static didn't make any sense, since we don't need it published as a file. I was considering adding a `_files` directory just for being able to `readFile` stuff, but resources.Get from /assets is a lot cleaner.